plugin/http/install/install.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Import to install OpenCensus HTTP tracing globally, replacing http.DefaultTransport
+package installhttpplugin
```

Comment:
I wonder if we can call it to defaulthttptrace

import _ "go.opencensus.io/plugin/http/defaulthttptrace"

Reply:
I wanted it to install the stats as well, when available.  What about "go.opencensus.io/plugin/http/instrumentdefault"?

Reply:
"go.opencensus.io/plugin/http/instrumentdefault" sounds good.

Reply:
How about `installer`? The path "/plugin/http" already speaks to the purpose of the import.